contactsummary/js/src/contact-summary/controllers/sickness.js

```diff
@@ -20,6 +20,7 @@ define([
         this.takenPreviously = 0;
         this.staffAverage = 0;
         this.ready = false;
+        this.currentPeriod = 'Unknown';
```

Comment:
Unless stated otherwise, please put a `-` (dash), so that we don't need to translate this string in the future